docs/specs/tripgo.swagger.yaml

```diff
@@ -2947,6 +2947,12 @@ definitions:
         required:
           - lat
           - lng
+      speed:
+        type: number
+        description: Speed of this vehicle at the provided location, in metres per second
+      distanceTravelled:
+        type: number
+        description: Distance travelled of this vehicle since start of the journey, in metres
```

Comment:
Just a comment: In the GTFS-R specs it is not clear if the “distance” is from the start of the journey or between stops. It is just the odometer of the vehicle but the specs is not clear where is the zero value.

Reply:
Ah, ok, then `distanceTravelled` is maybe not the best name and we should just name it `odometer`?

Reply:
Done here: https://github.com/skedgo/skedgo-java/pull/4316

Reply:
Yes. Let’s do that for now unless there is an update on the GTFS-R specs.